source/Outfit.cpp

```diff
@@ -65,6 +65,7 @@ namespace {
 		{"thrusting disruption", 0.},
 		{"thrusting slowing", 0.},
 		{"lateral thrust ratio", 0.},
+		{"lateral thrust ratio", 0.},
```

Comment:
```suggestion
		{"lateral thrust ratio", 0.},
```

Reply:
Nope, there does not. The fact that it is there suggests to me that I duplicated the line and intended to replace the label with something different but related, but didn't. 

I'm going to leave this open for now as a reminder to look into this and figure out what needs to be there, if anything.

Thanks for spotting it!

Reply:
Alright, this is fixed now. Thanks!